components/Modals/GroupNotifyMeModal/NotifyMeError.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useModalDispatch, hideModal } from 'providers/ModalProvider';
+
+import { Box, Button, Icon } from 'ui-kit';
+
+function NotifyMeSuccess(props = {}) {
```

Comment:
Could we quickly update the internal name of this component to prevent the chance for confusion later?

```suggestion
function NotifyMeError(props = {}) {
```